includes/plugin-instrumentation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Razorpay\Api\Api;
+use Razorpay\Api\Errors;
+
+class TrackPluginInstrumentation
+{
+    public $api;
+
+    public function __construct($key_id, $key_secret)
+    {
+        $this->api = new Api($key_id, $key_secret);
+    }
+
+    public function rzpTrackSegment($event, $properties)
+    {
+        try
+        {
+            if (empty($event) === true or
+                is_string($event) === false)
+            {
+                return ['status' => 'error', 'message' => 'event given as input is not valid'];
+            }
+
+            if (empty($properties) === true or
+                is_array($properties) === false)
+            {
+                return ['status' => 'error', 'message' => 'properties given as input is not valid'];
+            }
+
```

Comment:
@abdulwahidsharief Please include platform = 'WordPress', platform_version, woocommerce_version etc. Initiate these values within constructor and include to existing array $properties.

Reply:
@enishant done